client/scss/utilities/_root-scope-classes.scss

```diff
@@ -92,6 +100,14 @@
   text-decoration: none;
 }
 
+.plain-list {
+  @include plain-list();
```

Comment:
yeah, there's a couple of other instances where things could be refactored like the text-meta class. I'll do them in another PR